lib/cocoapods/installer.rb

```diff
@@ -626,19 +645,6 @@ def run_podfile_post_install_hook
 
     #-------------------------------------------------------------------------#
 
-    public
-
-    # @return [Array<Library>] The targets of the development pods generated by
-    #         the installation process.
-    #
-    def development_pod_targets
```

Comment:
unused method.

Reply:
FYI be careful when saying "unused method" - it might have been used by client scripts, not necessarily from within Cocoapods. This commit broke my scripting because I was depending on `development_pod_targets` being exposed.

Reply:
Likewise this broke our scripts. I think changes like this should go in the ChangeLog.

Reply:
@dnkoutso ^^ this might be more impactful for others that switch to 1.4.0

Reply:
Ill add it back and include a comment. Will be in rc.2

Reply:
Ill add it back and include a comment. Will be in rc.2